sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/ServerCall.java

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver;
+
+import com.azure.communication.callingserver.implementation.converters.PlayAudioConverter;
+import com.azure.communication.callingserver.implementation.models.PlayAudioRequest;
+import com.azure.communication.callingserver.models.CallRecordingStateResult;
+import com.azure.communication.callingserver.models.PlayAudioOptions;
+import com.azure.communication.callingserver.models.PlayAudioResult;
+import com.azure.communication.callingserver.models.StartCallRecordingResult;
+import com.azure.communication.common.CommunicationIdentifier;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+
+/**
+ * Sync Client that supports server call operations.
+ */
+public final class ServerCall {
```

Comment:
This files looks like a client. If so, we should name it `ServerCallClient` to comply to Java guidelines

Reply:
ServerCall has some service methods in it, but it is not presented here as a client.

Reply:
ServerCall is not presented as a client here as per our discussion previously.